src/utils.js

```diff
@@ -125,3 +125,8 @@ export function cssgu(strings, ...substitutions) {
       return isNaN(value) ? match : `${value * GU}px`
     })
 }
+
+export const iOS =
+  /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream
+
+export const isSafari = !!navigator.userAgent.match(/Version\/[\d.]+.*Safari/)
```

Comment:
Super nitpicky :haircut_man:

```suggestion
export const isSafari = /Version\/[\d.]+.*Safari/.test(navigator.userAgent)
```

Also should we export an object directly for these checks? e.g. `browser.ios`, `browser.safari`, etc.

Reply:
Yes, I agree. Thank your for clarifying it can be done in another PR :)